packages/engine/Source/Scene/GltfLoader.js

```diff
@@ -471,6 +475,22 @@ function processLoaders(loader, frameState) {
     ready = ready && metadataReady;
   }
 
+  const meshPrimitiveGpmLoader = loader._meshPrimitiveGpmLoader;
+  if (defined(meshPrimitiveGpmLoader)) {
+    const metadataReady = meshPrimitiveGpmLoader.process(frameState);
+    if (metadataReady) {
+      if (defined(loader._components.structuralMetadata)) {
+        oneTimeWarning(
+          "structural-metadata-gpm",
+          "Structural metadata is replaced with GPM data"
+        );
+      }
+      loader._components.structuralMetadata =
+        meshPrimitiveGpmLoader.structuralMetadata;
```

Comment:
The option of merging metadata (and their schemas) was considered, but cannot be addressed here. There are many caveats. An obvious one:

- The glTF contains `EXT_structural_metadata`, with a schema that has the `id : "MY_EXAMPLE_SCHEMA"`
- It also defines `NGA_gpm_local`. Metadata classes will be created for the PPE textures, and added to the schema.
- Will the schema afterwards _still_ have the `id : "MY_EXAMPLE_SCHEMA"`? 
  - Yes: This does not make sense, because it is a new schema, different from the original one
  - No: Which ID should it have? (And what will users think when their schema with `id: "MY_EXAMPLE_SCHEMA"` suddently disappears?)

Reply:
@javagl Does it make sense to document this limitation in an issue and add discussion there about potential solutions?